docs/topics/braintree.rst

```diff
@@ -227,6 +227,17 @@ solitude.
 
     :status 201: payment method created.
 
+Cancel a subscription. This will cancel the subscription in Braintree and is
+not reversible.
+
+.. http:post:: /braintree/subscription/cancel/
+
+    :<json string paymethod: the resource_uri of the subscription in solitude.
```

Comment:
Another possibility would be to do ``POST /braintree/subscription/123/cancel/`` which wouldn't require passing in a ``resource_uri``